src/plugins/console/public/lib/curl_parsing/curl.ts

```diff
@@ -79,12 +79,12 @@ export function parseCURL(text) {
     if (lines.length === 0) {
       return false;
     }
-    line = lines.shift().replace(/[\r\n]+/g, '\n') + '\n';
+    line = lines.shift()?.replace(/[\r\n]+/g, '\n') + '\n';
```

Comment:
so line could be undefined?

Reply:
.shift only returns undefined when lines array is empty and this condition was tested in lines above ```(lines.length === 0)```, I should change it to ! instead.